src/generator/AutoRest.Go/Extensions.cs

```diff
@@ -20,6 +20,10 @@ public static class Extensions
 
     public const string ReadOnlyConstraint = "ReadOnly";
 
+    private static readonly Regex IsApiVersionPattern = new Regex(@"^api[^a-zA-Z0-9_]?version", RegexOptions.IgnoreCase);
+
+    private static readonly Regex UnwrapAnchorTagsPattern = new Regex("([^<>]*)<a\\s*.*\\shref\\s*=\\s*[\'\"]([^\'\"]*)[\'\"][^>]*>(.*)</a>");
+
```

Comment:
nit, does it make sense to get this variables outside the func where they are used?